packages/react/.eslintrc.js

```diff
@@ -54,70 +75,75 @@ module.exports = {
       extends: [
-        'airbnb',
-        'airbnb-typescript',
+        'eslint:recommended',
         'plugin:jest/recommended',
         'plugin:react/recommended',
-        'plugin:react-hooks/recommended',
         'plugin:@typescript-eslint/recommended-requiring-type-checking',
-        'plugin:prettier/recommended',
-      ],
-      plugins: [
-        'react',
-        '@typescript-eslint',
-        'react-hooks',
-        'jest',
-        'prettier',
+
+        // extend last to override previous extensions
+        ...sharedExtensions,
       ],
+      plugins: ['@typescript-eslint', 'jest', 'react', 'react-hooks'],
+      settings: {
+        react: {
+          version: 'detect',
+        },
+      },
       rules: {
+        ...sharedRules,
+
+        // typescript eslint rules either not in recommended rule set or overridden
         '@typescript-eslint/explicit-module-boundary-types': 2,
         '@typescript-eslint/member-ordering': 'error',
         '@typescript-eslint/no-extra-semi': 'error',
+        '@typescript-eslint/no-floating-promises': ['off'],
         '@typescript-eslint/no-unused-expressions': [
           'error',
           { allowTernary: true },
         ],
-        '@typescript-eslint/no-floating-promises': ['off'],
+        '@typescript-eslint/no-use-before-define': ['error'],
         '@typescript-eslint/no-unused-vars': [
           'error',
           { argsIgnorePattern: '_', varsIgnorePattern: '_' },
         ],
         '@typescript-eslint/prefer-nullish-coalescing': 'error',
         '@typescript-eslint/restrict-template-expressions': ['off'],
         '@typescript-eslint/unbound-method': 'error',
+
+        // eslint rules either not in recommended rule set or overridden
         'comma-dangle': ['error', 'only-multiline'],
         'function-paren-newline': 'off',
         'generator-star-spacing': 'off',
         'global-require': 'off',
         'implicit-arrow-linebreak': 'off',
-        'import/no-cycle': 'off',
-        'import/no-extraneous-dependencies': ['off'],
-        'import/prefer-default-export': 'off',
-        'jest/expect-expect': ['error', { assertFunctionNames: ['expect*'] }],
-        'jest/no-mocks-import': 'off',
         'max-params': 2,
         'no-alert': 'error',
         'no-console': 'error',
+        'no-eval': 'error',
         'no-tabs': ['error', { allowIndentationTabs: true }],
+        'no-unused-vars': 'off', // prefer @typescript-eslint version
+        'prefer-const': 'error',
         'prefer-destructuring': ['error', { array: false, object: true }],
-        'prettier/prettier': ['error'],
+
+        // jest rules either not in recommended rule set or overridden
+        'jest/expect-expect': ['error', { assertFunctionNames: ['expect*'] }],
+        'jest/no-mocks-import': 'off',
+
+        // react rules either not in recommended rule set or overridden
+        'react/destructuring-assignment': ['error', 'always'],
+        'react/jsx-boolean-value': 'error',
```

Comment:
This doesn't appear to be the correct value for the rule?
https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-boolean-value.md

Reply:
You would think that but:

<img width="724" alt="image" src="https://user-images.githubusercontent.com/15002885/166831765-7b658a75-243e-4799-9705-0eebff8b0ed1.png">

In fact:
<img width="1407" alt="image" src="https://user-images.githubusercontent.com/15002885/166832171-83a586a1-4162-498e-8249-1f2b14616b89.png">

Reply:
lol, so I guess their docs are out of date.